transforms/input.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-env node */
+'use strict';
+
+let deprecate;
+
+try {
+  deprecate = require('ember-source/dist/ember.prod').deprecate;
+} catch(e) {
+  deprecate = require('../bower_components/ember/ember.prod').deprecate;
+}
```

Comment:
@rwjblue Could you review this now? I think the only interesting thing in the PR is trying to grab the correct ember. Not sure if we need to support the bower flow as I believe we are dropping ember 3.0+ from bower correct?